Sources/StreamChatUI/CommonViews/QuotedChatMessageView/QuotedChatMessageView.swift

```diff
@@ -270,8 +270,7 @@ open class QuotedChatMessageView: _View, ThemeProvider, SwiftUIRepresentable {
             )
             textView.text = nil
         } else {
-            voiceRecordingAttachmentQuotedPreview.isHidden = true
```

Comment:
This hiding of views is no longer needed?

Reply:
This is now handled by the unsupported view 👍